frontend/src/lib/components/EventPropertyTabs/eventPropertyFilteringLogic.ts

```diff
@@ -26,7 +24,8 @@ export const eventPropertyFilteringLogic = kea<eventPropertyFilteringLogicType>(
                             const isPostHogProperty = key.startsWith('$') && PROPERTY_KEYS.includes(key)
                             const isNonDollarPostHogProperty =
                                 isCloudOrDev && CLOUD_INTERNAL_POSTHOG_PROPERTY_KEYS.includes(key)
-                            return !isPostHogProperty && !isNonDollarPostHogProperty
+                            const isSystemProperty = props[key]?.system
```

Comment:
just include "is system property" here instead of having a separate button in the events table separate from the all the others 🤷